pkg/backup/backup.go

```diff
@@ -69,7 +69,17 @@ func (ac ActionContext) log(msg string, args ...interface{}) {
 type Action interface {
 	// Execute is invoked on an item being backed up. If an error is returned, the Backup is marked as
 	// failed.
-	Execute(ctx ActionContext, item map[string]interface{}, backup *api.Backup) error
+	Execute(ctx *backupContext, item map[string]interface{}, backup *api.Backup, backupper itemBackupper) error
```

Comment:
`backupContext` already has the `Backup` in it, so no need to pass the `Backup` in separately.

Reply:
I wonder if we should put the itemBackupper in the backupContext?

Reply:
good point on #1; maybe on #2. 

Does the move to eliminate ActionContext and just use backupContext within the actions seem reasonable? I was going to have to add many of the fields from the latter to the former otherwise, so this seemed like a better choice.

Reply:
Yeah I'm fine getting rid of the ActionContext and replacing with backupContext